app/src/components/widgets/charts/investigatedTrendChart/investigatedTrendChart.jsx

```diff
@@ -65,6 +85,16 @@ export class InvestigatedTrendChart extends Component {
       this.props.observer.unsubscribe('widgetResized', this.resizeChart);
   }
 
+  onChartClick = (data) => {
+    const { widget, statisticsLink } = this.props;
+    const id = this.isTimeline
```

Comment:
Be careful with timeline mode. In this case user should be redirected to the launches page with new filter (according to our 4th version).